java/src/org/openqa/selenium/grid/node/local/LocalNode.java

```diff
@@ -471,6 +477,49 @@ public HttpResponse executeWebDriverCommand(HttpRequest req) {
     return toReturn;
   }
 
+  @Override
+  public HttpResponse downloadFile(HttpRequest req, SessionId id) {
+    // When the session is running in a Docker container, the download file command
+    // needs to be forwarded to the container as well.
+    SessionSlot slot = currentSessions.getIfPresent(id);
+    if (slot != null && slot.getSession() instanceof DockerSession) {
+      return executeWebDriverCommand(req);
+    }
+    if (this.downloadsDir.isEmpty()) {
+      throw new WebDriverException(
+        "Please specify the directory that would contain downloaded files and restart the node.");
```

Comment:
It is better to tell the user how to do that, ideally to point them to the docs link.

Reply:
Perhaps a brief version of the instructions is fine. Later the link can be added.

Reply:
Cool. Updating it now.